src/components/Album.jsx

```diff
@@ -7,7 +7,7 @@ export const Album = ({ album }) => {
 
   return (
     <div className="album-container">
-            <CoverImage img={album.images[1].url} alt={album.name} />
+      <CoverImage img={album.images[0].url} alt={album.name} />
```

Comment:
let's change this to something like ```album.images.find(image => image.height === 640).url```

It's better to be a bit more explicit when we're using data from an external site and assume that they can change what their API returns without warning. (I know we're using fake data here, but this is good practice in the future).

Reply:
I made the change but added it to a variable. I'm not sure which is a better practice?